src/host/ConsoleArguments.cpp

```diff
@@ -21,6 +21,79 @@ const std::wstring_view ConsoleArguments::INHERIT_CURSOR_ARG = L"--inheritcursor
 const std::wstring_view ConsoleArguments::FEATURE_ARG = L"--feature";
 const std::wstring_view ConsoleArguments::FEATURE_PTY_ARG = L"pty";
 
+std::wstring escape_argument(std::wstring_view ac)
+{
+    if (ac.empty())
+    {
+        return L"\"\"";
+    }
+    bool hasspace = false;

```

Comment:
```suggestion
    bool hasSpace = false;
```

Please camel case. This looked like "hass pace" to me.